tests/sniffTest

```diff
@@ -68,15 +68,13 @@ if [[ $rc != 0 || -s err ]]; then
 	echo Output: $(cat out)
 	exit 1
 fi
-$RUNC kill c1
-rm -rf /run/opencontainer/containers/c1
```

Comment:
There are timing issues with kill then delete... need to wait for the detached container to be in the destroyed state before killing it.  We could do the sleep 6, then delete c1 here if you prefer instead of the end of the script or use a smaller sleep in the tests then a smaller sleep after to wait for the windows to delete the detached containers.